### PR TITLE
[FIX] display_import_button: fixed add button visibility.

### PR DESCRIPTION
**Pull request by AndreaGeraldo**
<h2>RELATED</h2>

* [TASK#62637](https://www.vauxoo.com/web#id=62637&model=project.task&view_type=form&cids=1&menu_id=)

* [TICKET 1#20365](https://www.vauxoo.com/web#cids=1&id=20365&menu_id=&model=helpdesk.ticket&view_type=form)

* [TICKET 2#16382](https://www.vauxoo.com/web#active_id=20365&cids=1&id=16382&menu_id=&model=helpdesk.ticket)

* [RELATED MR](https://git.vauxoo.com/vauxoo/nortenas/-/merge_requests/147)

* [SOURCE display_import_button MODULE ](https://github.com/GlorieuxAmichia/display_import_button)

<h2>OVERVIEW</h2>

On the Contacts app, after selecting a given contact and clicking on the Edit button, the Add button that's normally displayed under the Contacts & Addresses notebook page, is missing.

Example images can be found on TICKET 2#16382.

<h2>LISTED FEATURES</h2>

The implementation of the module **display_import_button** was causing this issue. The inheritance done to KanbanView.buttons on display_import_button/static/src/xml/template.xml had a t-if whose condition was not being met and therefore it would not display the Add button.

The previously mentioned module overrides the behavior of [kanban_controller.js](https://github.com/Vauxoo/odoo/blob/12.0/addons/web/static/src/js/views/kanban/kanban_controller.js) and while adding its own set of conditions to hide the import/create buttons from certain views, it also affected the rendering of other buttons.

To fix this, a [new](https://github.com/Vauxoo/display_import_button/pull/1/files#diff-20c08088c560a6feebe76c823b181312bc74178a5846d93c37106c2536f632f6R14) key-value pair has been added to the rendering context of the KanbanView.buttons view. Said False value is used to meet the conditional statement [on template.xml](https://github.com/Vauxoo/display_import_button/pull/1/files#diff-b1ab611e58f6e69d20690f8b77cf702c55a97c22b79b50c7747f9ee6f7cd8b93R22).

<h2>TESTING</h2>

* [INSTANCE](https://t62637-002-nortenas120.dev.nortenas.deployv.com/)

### CODE REVIEW

File: display_import_button/__manifest__.py [hunk 1]
```diff
@@ -7,13 +7,14 @@
 		This module is used to manage the display of the "import" button on your
 		list, form, kanban view, according to your needs.
 	""",
-	'author': "SLife Organization",
+	'author': "SLife Organization, Amichia Fréjus Arnaud AKA",
+	"website": "https://slifeorganization.com",
 	'category': 'web',
-	'version': '1.0',
+	'version': '1.1',
```
Comment (imanie383) [inline display_import_button/__manifest__.py]: image

https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#version-numbers

File: display_import_button/static/description/index.html [hunk 2]
```diff
@@ -28,7 +28,17 @@ <h3 class="oe_slogan">Use To Hide Import Button</h3>
     </div>
 </section>
 
-<section class="oe_container oe_dark">
```
Comment (imanie383) [inline display_import_button/static/description/index.html:31]: Remove all .DS in the branch

image

image

**Comment by imanie383** [comment]
Add the commit documentation

**Comment by imanie383** [comment]
apply the pre-commit to standardize the "

**Comment by imanie383** [comment]
Technically Ok +1

**Comment by hugho-ad** [comment]
sorry, why is this targeting different branch than 12.0?

**Comment by hugho-ad** [comment]
@AndreaGeraldo 

pls, create the PR targeting branch 12.0 norteñas is the only repo using this module, infact, I would rather avoid this and move this module to the client's repo

**Comment by imanie383** [comment]
> sorry, why is this targeting different branch than 12.0?

Because 
the version that norteñas uses and Vauxoo/display_import_button/12.0 are different
If we merge into 12 and some other project uses that branch, we will affect them, so we decided to create a branch for norteñas

**Comment by hugho-ad** [comment]
@imanie383 

Norteñas is the only project using this

better move the module to the clients repo to avoid maintaining this

**Pull request merged**